getting-started/download.md

```diff
@@ -1,43 +1,70 @@
-| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
-| [WolvenKit Nightly](https://github.com/WolvenKit/WolvenKit-nightly-releases/) | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases)](https://github.com/WolvenKit/WolvenKit-nightly-releases/releases/latest) | ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly) |
-| [WolvenKit](https://github.com/WolvenKit/WolvenKit/)                          | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit)](https://github.com/WolvenKit/WolvenKit/releases/latest)                                   | ![GitHub branch checks state](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only)    |
-
-The latest GitHub WolvenKit release will be marked on the left column with green text. Expand the **Assets** tab to view available downloads for any given WolvenKit release.
+The recommended download is the WolvenKit Nightly installer
+
+<table>
+  <tr align="left">
+    <th>Source Code</th>
+    <th>Installer</th>
+    <th>Checks</th>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        WolvenKit Nightly
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases" alt="GitHub nightly release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly" alt="GitHub branch checks">
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/">
+        WolvenKit Stable
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/releases/latest">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit" alt="GitHub stable release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only" alt="GitHub branch checks">
+    </td>
+  </tr>
+</table>
 
 {% embed url="https://www.nexusmods.com/cyberpunk2077/mods/2201" %}
 
-### WolvenKit
+## System Requirements
 
-The main WolvenKit application in a portable zip file (Recommended)
```

Comment:
The portable WolvenKit is the recommended (and only) way to install the application. Should not be removed.

Reply:
I think we're both referring to the ZIP file that contains the compiled WolvenKit program? I'm unsure how many people know what a "portable" is, this is the first time I've heard it in this context.